lib/webhooks/__tests__/utils.ts

```diff
@@ -23,7 +23,7 @@ export function getTestExpressApp() {
 
 export function headers({
   hmac = 'fake',
-  topic = 'PRODUCTS_CREATE',
+  topic = 'products/create',
```

Comment:
In the description of the PR you are saying : "we should be using graphqlTopic to show the PRODUCTS_UPDATE format rather than products/update", but here it seems that you're doing the opposite??